docs/en/enterprise-edition/content-collections/runtime-security/agentless-scanning/configure-accounts/frag-start-agentless-scan.adoc

```diff
@@ -1,27 +1,65 @@
-[#start-agentless-scan]
+[#agentless-scanning-for-cloud-accounts]
 [.task]
-== Start an Agentless Scan
+=== Agentless Scanning for Cloud Accounts
 
-Agentless scans start immediately after onboarding the cloud account.
-By default, agentless scans are performed every 24 hours, but you can change the interval on the *Manage > System > Scan* page under *Scheduling > Agentless*.
+When new cloud accounts are onboarded to Prisma Cloud with the "Agentless Scanning" option enabled, scanning starts immediately. If this option is disabled before onboarding, scanning does not occur for those accounts.
+
+When you enable agentless scanning for existing accounts in Prisma Cloud, it does not trigger an immediate scan. Instead, the account is included in the next scan cycle, which occurs every 24 hours by default.
+
+To modify the scan cycle, see <<modifying-the-agentless-scan-interval>>
+
+[#editing-agentless-scan-settings]
+=== Editing Agentless Scan Settings
+
+For existing cloud accounts that have agentless scan settings disabled, you can now modify the settings directly in Runtime Security > Manage > Cloud accounts page. 
+
+To edit agentless scan settings, complete the following steps:
+
+[.procedure]
+. Go to *Runtime Security > Manage > Cloud accounts*.
+
+. Click the *Edit Account* icon from the Actions column for the account. 
+
+. In the *Account and Agentless setup* dialog box, go to *Agentless scanning* section.
+
+. Modify the agentless configuration options in this section. 
+
+ . Click *Save*.
+
+After the configuration is modified, the next scan uses the updated settings.
+
+[#modifying-the-agentless-scan-interval]
+[.task]
+=== Modifying the Agentless Scan Interval
+
+By default, agentless scans are triggered every 24 hours.
+To change the interval, complete the following steps.
+[.procedure]
+. Go to *Runtime Security > Manage > System*. 
+. Click the *Scan* tab. 
+. In the *Scheduling* section, in *Agentless* box, type the new duration for the scan cycle.
+. Click *Save*
 
 image::runtime-security/agentless-interval.png[width=800]
 
+[#manually-starting-an-agentless-scan]
+[.task]
+=== Manually Starting an Agentless Scan
 To manually start a scan, complete the following steps.
 
 [.procedure]
 
 . Go to *Runtime Security > Manage > Cloud accounts*.
 
-. Click the scan icon on the top right corner of the accounts table.
+. Select the cloud account you want to scan. 
```

Comment:
this isn't accurate - the selection of accounts does not effect which accounts are scanned, the button scans all accounts regardless of the selection.

Reply:
I have removed this sentence. I have added a note to the previous step "Click the Scan Options icon..." the following sentence: 
Note: Scanning starts for all the accounts that have the agentless scanning option enabled.